README.md

```diff
@@ -1,6 +1,7 @@
-Geddy-Passport
+# Geddy-Passport
 
-* * *
+## DEPRECATION NOTICE
+**Geddy is no longer actively maintained. It is not recommended for new project.**
```

Comment:
It sounds picky of me @phanect, but it is better English to say:
"Geddy is no longer actively maintained, and therefore it is not recommended to be used for any new projects."